UITests/Responses/TransferFunds/TransferBelowTransactionLimitError.json

```diff
@@ -1,6 +1,6 @@
 {
     "errors" : [ {
-        "message" : "Requested transfer amount $0.01, is below the transaction limit of $1.00.",
+        "message" : "Requested transfer amount $0.10, is below the transaction limit of $1.00.",
```

Comment:
instead of updating the asserts you have to check why it's happening and what should be the correct behaviour

Reply:
Sure Jaspreet